oc_cdtapi/JenkinsAPI.py

```
@@ -359,7 +364,7 @@ def _update_build_status(self):
         build_path = posixpath.join("job", self.job_name, str(self.build_number), 'api', 'json')
         response = self.jenkins_client.get(build_path)
         if response.status_code == 404:
-            raise HttpAPIError(code=404, text=f"No build exist at {build_path}")
+            raise JenkinsError("No build exist at %s" % build_path)
```

Ruff (0.15.9)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)